terraform/deployments/cluster-services/templates/alertmanager-config.tpl

```diff
@@ -21,6 +21,14 @@ alertmanager:
         group_interval: 12h
         active_time_intervals:
         - inhours
+      - match:
+          alertname: MirrorFreshnessAlert
+        receiver: 'slack-mirror-freshness'
+        repeat_interval: 1d
+        group_wait: 12h
+        group_interval: 12h
+        active_time_intervals:
+        - inhours
```

Comment:
optional: we probably wanna do this via a label so we don't have to configure individual alerts in the alertmanager config. No probs if you wanna leave that for another day though. Same for the Slack bit I guess.

Reply:
Yah agree - I think there's defo a much nicer way to configure these alerts more generally. I'm going to merge this in for now, so we have something as I'm running out of time. But on my to do list to revisit this config.